QnA/bad_question/qna_2023-12-08_20.json

```diff
@@ -0,0 +1,10 @@
+{
+    "score": 0.5330233619426845,
+    "crawl_id": "24232c84-bffb-441b-a756-97e5e338ae23",
+    "chunk_id": "35e08f03-05f7-484a-af10-606a36b125c8",
+    "title": "Product photos",
+    "url": "https://inspection.canada.ca/food-recall-warnings-and-allergy-alerts/2018-06-02/eng/1527975832796/1527975835551",
+    "text_content": "Product photos Printer ready version of photos",
+    "question": "",
```

Comment:
why is this empty

Reply:
Because something went wrong, this is why it's classified in `bad_question`